include/cpuinfo.h

```diff
@@ -386,6 +386,9 @@ enum cpuinfo_uarch {
 	/** Intel/Marvell XScale series. */
 	cpuinfo_uarch_xscale = 0x00100600,
 
+	/** Intel Meteor Lake microarchitecture (Core Ultra 1st gen) */
+	cpuinfo_uarch_meteor_lake = 0x00100700,
```

Comment:
2 things:  move this after sunny_cove & consider using the P and E core microarchitectures
meteorlake is the SoC
redwood and crestmont are the microarchitectures
each core can have a different microarchitecture and in XNNPack we can have a different microkernel for each thread.

Reply:
Thanks for the suggestions!
I noticed both P and E cores of MTL have same model number. I'll find some other field inside cpuid to identify the P core and E cores